api/checks/update.go

```diff
@@ -263,9 +261,9 @@ func getDiffSummary(aeAPI shared.AppEngineAPI, diffAPI shared.DiffAPI, suite sha
 				data.More++
 			}
 		}
-		if checksCanFailAndPass {
-			failure := "failure"
-			data.CheckState.Conclusion = &failure
+		if checksCanBeNonNeutral {
+			actionRequired := "action_required"
```

Comment:
I'm happy to try this and see how it works in practice. I'd expect it to occasionally be a nuisance that the same action is required on as many checks as we have browsers. This could be changed back to "neutral" if a quickfix for that or other problem isn't possible.

Reply:
As many checks as we have _failing_ browsers, to be precise.